wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -28,10 +28,10 @@ class WagnGenerator < Rails::Generators::AppBase
   
 ## should probably eventually use rails-like AppBuilder approach, but this is a first step.  
   def dev_setup
-    @gem_path = options['gem-path']
+    @gem_path = ENV['WAGN_DEV_GEM_PATH'] || options['gem-path']
```

Comment:
no, this isn't what I was suggesting.  We want the Gemfile to have something like:
````
  gem 'wagn', :path=>"ENV['WAGN_GEM_DEV_PATH']"
````
the point is to be able to have a standard path in your profile but set a different one whenever you run a server, tests, etc.